source/core-xml-schemas/CHANGELOG.md

```diff
@@ -11,6 +11,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+## [core-xml-schemas-v3.0]
```

Comment:
Don't normally add a release heading and diff to the changelog till we are ready to release.

Reply:
When we do release you will need to add an entry to the appropriate markdown table in the root readme so we have a link to the release zip

Reply:
Fine.  I've left the additions in the CHANGELOG within the UNRELEASED section for time being.